assessment.md

```diff
@@ -93,9 +93,50 @@ Project Management is Covered by Team 2
 ### Interactions
 
 * Interaction with the customer
+  * :disappointed: Disregards the customer's feelings and
+  refuses to take their needs into account
+  * :confused: Regards some of the customer's feelings
+  but fails to take their ideas and needs into account
+  * :neutral_face: Regards most of the customer's feelings
+  and takes some of their ideas and needs into account
+  * :smiley: Regards all of the customer's feelings and
+  incorporates most of their ideas and needs while also communicating challenges
+  * :sunglasses: Regards all of the customer's feelings, improves ideas,
+  incorporates all of their needs, clearly communicates challenges and
+  provides updates on progress
+
 * Interaction with the team leaders
+  * :disappointed: Impolite, impatient and does not respond to team leaders
+  * :confused: Polite yet impatient and is partially responsive to
+  team leaders but does not communicate clearly
+  * :neutral_face: Polite, patient and is mostly responsive to team leaders
+  with moderate communication
+  * :smiley: Polite, patient and is responsive to team leaders with clear
+  communication
+  * :sunglasses: Polite, patient and is responsive to team leaders with clear
+  communication, providing multiple progress updates
+
 * Interaction with the course instructor
+  * :disappointed: Impolite, impatient and does not communicate with the course
+  instructor
+  * :confused: Impolite, impatient and only communicates with the course
+  instructor when personally addressed
+  * :neutral_face: Polite yet impatient and does not communicate clearly with the
+  course instructor
+  * :smiley: Polite, patient and communicates with the course instructor
+  * :sunglasses: Polite, patient and communicates with the course instructor to
+  engage in helpful learning experiences
+
 * Interaction with the student technical leaders
+  * :disappointed: Impolite, impatient and does not communicate with the technical
+  leaders
+  * :confused: Impolite, impatient, does not speak to the technical leaders when
+  necessary and communicates poorly
```

Comment:
Note that there is a small phrasing mistake in these two points since the first one does have an "and" and the second one does not but uses an extra comma. What is the standard? Can you quickly rephrase these  points and any others like them?

Reply:
I worked on standardizing phrasing for the Interactions subsections through making each subsection of Interactions a standard format. Please let me know if there is more that I can do to standardize or rephrase!